js/script.js

```diff
@@ -151,7 +155,7 @@ const success = () => {
   error.classList.add('deactive-error');
 };
 
-submitBtn.addEventListener('click', (event) => {
+submitBtn.addEventListener('click', event => {
   const pattern = /[A-Z]/;
   if (pattern.test(emailInpute.value)) {
     failed(event);
```

Comment:
- [ ] The form validation still doesn't work. I submitted a form with an email `ex: Amine@gmail.com` and the form was submitted successfully which It's supposed to show an error next to the submit button. Kindly check the requirements again:
- The contact form contains at least one validation checking if the email is written in `all lowercase`:
  - If a user submits the contact form with the email field containing uppercase characters an error message should show up.
  - If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

Reply:
yes, you right.
I'm fix the error